api/v1/controllers/subjects.js

```diff
@@ -535,9 +535,6 @@ module.exports = {
    * DELETE /v1/subjects/{key}/relatedLinks/
    * DELETE /v1/subjects/{key}/relatedLinks/{name}
    *
-   * Deletes specified/all related links from the subject and sends updated
```

Comment:
Already reverted that, did this just to check a new branch as a copy of master works on Travis